### ISSUE TITLE
make DefaultChannelModes configurable

### ISSUE THREAD

**Issue by slingamn** [oragono/oragono#132]
In my deployment, I want to allow any user to set the channel topic; right now `OpOnlyTopic` (channel mode `+t`) is default. It would be nice if this could be set in the YAML config?

**Comment by DanielOaks** [comment]
I could see this for sure, yeah. I'd probably just have it as a string like `"+nt"` under `channels:`, maybe along the lines of:
```
# channel options
channels:
    # modes that are set when new channels are created
    # +n is no-external-messages and +t is op-only-topic
    # see  /QUOTE HELP cmodes  for more channel modes
    default-modes: +nt

    # channel registration - requires an account
    registration:
        # can users register new channels?
        enabled: true
```

We already parse modes under the opers section [here](https://github.com/oragono/oragono/blob/master/oragono.yaml#L199), so we can prolly just clone however that does it, and have DefaultChannelModes as a var under Server or similar.

**Comment by DanielOaks** [comment]
Implemented by #133, thanks @slingamn!
